dev-requirements.txt

```
@@ -1 +1,2 @@
 tox
+semver
```

Updating to a beta version adds type hints which fixes one of the `# type: ignore` comments in `version.py`
```suggestion
semver==3.0.0.dev3
```